app-misc/peaclock/peaclock-0.1.3.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A colourful binary clock for the terminal"
+DESCRIPTION="Features:
+* digital clock
+* binary clock
+* 12/24 hour time
+* customize with hex colour codes
+* set a custom unicode character for the binary clock graphic
+* compact or expanded mode
+* toggle visibility of digital and binary clocks
+* command prompt with a readline-like interface"
+HOMEPAGE="https://github.com/octobanana/peaclock"
+COPYRIGHT="2019 Brett Robinson"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="0041de0e50429786dff4152c60f6dbd84249287d289ad6d7573e47c9f9b62afc"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+commandBinDir=$binDir
+if [[ $targetArchitecture == x86_gcc2 ]]; then
+SECONDARY_ARCHITECTURES="x86"
+commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	$portName = $portVersion
+	cmd:$portName
```

Comment:
This didn't work for me (got a policy error for no matching provides for /bin/peaclock), changing it to "cmd:peaclock" worked for me